scripts/west_commands/run_common.py

```diff
@@ -222,8 +222,8 @@ def do_run_common(command, args, runner_args, cached_runner_var):
     runner_cls.add_parser(parser)
     parsed_args, unknown = parser.parse_known_args(args=final_runner_args)
     if unknown:
-        raise CommandContextError('Runner', runner,
-                                  'received unknown arguments', unknown)
+        raise CommandContextError(textwrap.dedent("""\
```

Comment:
I agree this is a bug, but CommandContextError only takes an optional returncode argument, not a string. For now, how about just deleting the argument?

Reply:
Yes, that would work, but especially in this situation it would be good for the user to see which argument was flawed. Maybe a print on err would be suitable?